### PR TITLE
HdfsWordCount example

### PR DESCRIPTION
**Pull request by tawan0109**
* Add a Mobius **streaming** example `HdfsWordCount`
* Spark `HdfsWordCount` example is available at https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/streaming/HdfsWordCount.scala

### CODE REVIEW

File: examples/Examples.sln [hunk 1]
```diff
@@ -11,6 +11,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SparkXml", "SparkXml\SparkX
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EventHub", "EventHub\EventHub.csproj", "{DF840BFB-B3A3-493D-B667-4CF21ADBFCAD}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HdfsWordCount", "HdfsWordCount\HdfsWordCount.csproj", "{6A2C7CF9-D64E-490D-9841-269EE14F7932}"
```
Comment (skaarthik) [inline examples/Examples.sln]: Consider moving this example to a folder named "streaming" under examples. We also need to move other examples to respective sub-folders but that can be done later.

File: examples/HdfsWordCount/HdfsWordCount.csproj [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{6A2C7CF9-D64E-490D-9841-269EE14F7932}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.Spark.CSharp.Examples</RootNamespace>
+    <AssemblyName>HdfsWordCount</AssemblyName>
```
Comment (skaarthik) [inline examples/HdfsWordCount/HdfsWordCount.csproj]: Consider following same naming convention as other examples

**Comment by skaarthik** [comment]
Add instructions to https://github.com/Microsoft/SparkCLR/blob/master/notes/running-sparkclr-app.md#running-examples-in-local-mode

File: examples/Examples.sln (hunk 1 above)
Comment (tawan0109) [inline examples/Examples.sln]: Moved.

File: examples/HdfsWordCount/HdfsWordCount.csproj (hunk 2 above)
Comment (tawan0109) [inline examples/HdfsWordCount/HdfsWordCount.csproj]: Renamed to `SparkClrHdfsWordCount`

**Comment by tawan0109** [comment]
Added.

**Comment by skaarthik** [comment]
LGTM

**Pull request merged**